xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -323,9 +315,7 @@ CDVDVideoCodecAndroidMediaCodec::CDVDVideoCodecAndroidMediaCodec(CProcessInfo &p
 , m_opened(false)
 , m_jnivideoview(nullptr)
 , m_jnisurface(nullptr)
-, m_crypto(nullptr)
```

Comment:
m_crypto is nullptr checked in dtor, I'd keep the nullptr initialization.

Reply:
It's initialzed in .h file

Reply:
Thanks! That's even better. I overlooked that.